### PR TITLE
Remove old install docs

### PR DESCRIPTION
**Pull request by Girgias**
This should cover the instal/ folder with #661

### CODE REVIEW

File: install/fpm/configuration.xml [hunk 1]
```diff
@@ -882,7 +882,7 @@ php_admin_value[memory_limit] = 32M
      cannot be overridden with <function>ini_set</function>.
     </para>
     <para>
-     As of 5.3.3, PHP settings are also possible to be set in webserver.
+     PHP settings can also possible to be set in the webserver.
```
Comment (jhdxr) [inline install/fpm/configuration.xml:885]: hmm, I think it should be `PHP settings can be set in the webserver.` if you look at grammar.

File: install/pecl.xml [hunk 2]
```diff
@@ -332,7 +341,7 @@ $ pecl install extname-0.1
    Sometimes, using the <literal>pecl</literal> installer is not an option.
    This could be because you're behind a firewall, or it could be because the
    extension you want to install is not available as a PECL compatible
-   package, such as unreleased extensions from SVN.  If you need to build such
+   package, such as unreleased extensions from git or SVN.  If you need to build such
```
Comment (jhdxr) [inline install/pecl.xml:344]: Git

**Review by jhdxr — CHANGES_REQUESTED** [review]

File: install/fpm/configuration.xml (hunk 1 above)
Comment (cmb69) [reply install/fpm/configuration.xml:885]: Much better, but stills feelsnot quit right. Maybe "PHP settings can be set in the webserver configuration"?

Comment (afilina) [inline install/fpm/configuration.xml:885]:
```suggestion
     PHP settings can be set in the webserver configuration.
```

**Comment by afilina** [comment]
@Girgias The module identifier was renamed in PHP 8, so I resolved the merge conflict to `php_module` instead of `php8_module`. I also added the comment as a mergeable suggestion, so after this is done, the PR looks ready to go.

File: install/pecl.xml [hunk 3]
```diff
@@ -331,7 +340,7 @@ $ pecl install extname-0.1
...
```
Comment (mumumu) [inline install/pecl.xml:343]:
```suggestion
   package, such as unreleased extensions from git.  If you need to build such
```

**Comment by Girgias** [comment]
> > I also added the comment as a mergeable suggestion, so after this is done, the PR looks ready to go.
> 
> +1. If no one is opposed to merge this PR for a while, I want to commit suggestions and process this PR.
> It is not good for PHP users, especially for newbies to leave this PR as it is.

Sorry forgot about this PR, but feel free to apply suggestions and commit. :-)

**Pull request merged**